### PR TITLE
Impling: Suffix Toggle

### PR DESCRIPTION
**Pull request by basicDamen**
Adds toggle to enable/disable 'impling' suffix after corresponding impling name. Requested in #4041.
impling_with_suffix
impling_without_suffix

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingMinimapOverlay.java [hunk 1]
```diff
@@ -73,7 +73,16 @@ public Dimension render(Graphics2D graphics)
 			if (config.showName())
 			{
 				Point textLocation = new Point(impLocation.getX() + 1, impLocation.getY());
-				OverlayUtil.renderTextLocation(graphics, textLocation, imp.getName(), color);
+
+				if (config.showNameSuffix())
+				{
+					OverlayUtil.renderTextLocation(graphics, textLocation, imp.getName(), color);
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingMinimapOverlay.java]: Don't duplicate logic for a changed string.

```
String impName = imp.getName();
if (!config.showNameSuffix())
{
    impName = impName.substring(0, impName.indexOf(" "));
}

OverlayUtil.renderTextLocation(graphics, textLocation, impName, color);
```

File: runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingsOverlay.java [hunk 2]
```diff
@@ -138,7 +138,15 @@ private void drawImp(Graphics2D graphics, Actor actor, String text, Color color)
 		Point textLocation = actor.getCanvasTextLocation(graphics, text, actor.getLogicalHeight());
 		if (textLocation != null)
 		{
-			OverlayUtil.renderTextLocation(graphics, textLocation, text, color);
+			if (config.showNameSuffix())
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingsOverlay.java]: Same as above.

```
if (!config.showNameSuffix())
{
    text = text.substring(0, text.indexOf(" "));
}
...
```

File: runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingMinimapOverlay.java [hunk 3]
```diff
@@ -73,7 +73,16 @@ public Dimension render(Graphics2D graphics)
 			if (config.showName())
 			{
 				Point textLocation = new Point(impLocation.getX() + 1, impLocation.getY());
-				OverlayUtil.renderTextLocation(graphics, textLocation, imp.getName(), color);
+
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingMinimapOverlay.java]: Remove this newline?

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingMinimapOverlay.java]: You may have removed the wrong one in 8162820? I think that change is fine also, but I suggested this to keep variable assignment separated from other function calls, and not have random line breaks separating variables.

Comment (basicDamen) [reply runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingMinimapOverlay.java]: I couldn't figure out which line you're talking about? I scanned it and didn't see any white space, except the one I already submitted a commit for.

**Pull request closed without merging**